### ISSUE TITLE
Enable Synology Chat webhook support

### ISSUE THREAD

**Issue by samsam-rolon** [containrrr/shoutrrr#162]
**Is your feature request related to a problem? Please describe.**
Slack webhooks are not working with synology chats webhooks

**Describe the solution you'd like**
Supporting Synology Chat webhooks

**Describe alternatives you've considered**
Telegram would be great :)

**Additional context**
https://www.synology.com/en-global/knowledgebase/DSM/tutorial/Collaboration/How_to_configure_webhooks_and_slash_commands_in_Chat_Integration

**Comment by piksel** [comment]
> **Describe alternatives you've considered**
> Telegram would be great :)

Telegram is supported! Use [`shoutrrr` notification](https://containrrr.dev/watchtower/notifications/#containrrrshoutrrr) with the [telegram service](https://containrrr.dev/shoutrrr/services/telegram/).

**Comment by samsam-rolon** [comment]
Intresting, I still prefer Synolgy Chat :-)
And I will deploy shoutrrr :)

**Comment by lucafolin** [comment]
is Synology Chat supported now ? I am unable to get it working. It would be great if Synology Chat works

**Comment by piksel** [comment]
No. I don't think anyone has been working on it. Since it's proprietary and closed source it's not something I would personally work on, but if someone wants to add it we're open to Pull Requests.
